engine/gamesys/wscript

```diff
@@ -40,8 +40,10 @@ def configure(conf):
         pass
     elif re.match('arm.*?android', platform):
         conf.env.append_value('LINKFLAGS', ['-lEGL', '-lGLESv1_CM', '-lGLESv2', '-landroid'])
-    elif platform == "linux" or platform == "x86_64-linux":
+    elif platform == "x86_64-linux":
         conf.env['LIB_X'] = ['Xext', 'X11', 'Xi', 'GL', 'GLU']
+    elif platform == "arm64-linux":
+        conf.env['LIB_X'] = ['Xext', 'X11', 'Xi', 'EGL', 'GLESv1_CM', 'GLESv2']
```

Comment:
Ok for tests, and I know it's old code, but the build artefact should probably use `use = "X OPENGL"` and set those libraries into the LIB_OPENGL variable instead (as you did in the wscript above)